irods/version.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-__version__ = '1.1.9'
+__version__ = '1.2.0.dev3'
```

Comment:
will change this before merging.  to just 1.2.0 i think... or can leave this here until we're closer to release date.

Reply:
yes, okay, i'll go back to 1.1.9 for this PR.  just had to bump it to be able to push to pypi.